apps/darts-modernisation/darts-api/test.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       ingressHost: darts-api.test.platform.hmcts.net
       environment:
         DARTS_GATEWAY_URL: https://darts-gateway.test.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts.test.apps.hmcts.net
```

Comment:
This is already set via the main chart, I double-checked, see below.

```
√ darts-api DMP-2005-event-logging % k -n darts-modernisation describe po darts-api-java-6f46b4fbc6-c5tt6 | grep DARTS_PORTAL_URL
      DARTS_PORTAL_URL:                   https://darts.test.apps.hmcts.net
```

Reply:
ok, so we can remove these URLs from all flux config?

Reply:
No, some of them need to be there due to the value of the `.Values.global.environment` variable that is used by the charts. For example, for staging the value is `stg` which would produce an incorrect value of `https://darts.stg.apps.hmcts.net` for `DARTS_PORTAL_URL`.

Reply:
Could be removed from `demo` but the others are required. Tbh it does no harm, just won't make any difference.